Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -185,7 +172,8 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         guard baseURL.chuzzle() != nil else { fatalError() }  //TODO
 
         // Compute the actual input file path.
-        let path: AbsolutePath = isDirectory(inputPath) ? inputPath.appending(component: Manifest.filename) : inputPath
+        let path: AbsolutePath = isDirectory(inputPath) ?
+            Manifest.versionSpecificManifest(path: inputPath, fileSystem: fileSystem) : inputPath
```

Comment:
I think we should just remove this and use the `inputPath` directly now, i.e., it is illegal to provide a directory path to this method.